### PR TITLE
Leather/bone/chitin recipe staggering, armguard split

### PR DESCRIPTION
**Pull request by chaosvolt**
This is a series of changes to the levels certain body-material armors are learned. The main reason was a number of inconsistencies, and additionally a recipe gap at level 5. Now go forth and be a proper monster hunter. o3o

| Level | Leather/Fur | Bone | Chitin |
| --- | --- | --- | --- |
| 2 | Gauntlets, Vambraces, Boots |  |  |
| 3 | Armguards | Gauntlets, Boots |  |
| 4 | Helm | Armguards | Gauntlets, Boots |
| 5 | Leather Armor, Fur Armor | Helm | Armguards |
| 6 |  | Armor | Helm |
| 7 |  |  | Armor |

Item additions/changes:
- Leather vambraces, as a proper armguard-equivalent for leather gear, being low coverage yet low encumbrance.
- Split leather and bone body armor, resulting in the addition of arm guards for both types. These are larger than most armguards, and thus match the coverage, encumbrance, etc that the arm protection of the old suits would have.
- Altered recipe requirements, value, volume, and weight of leather/body body armor accordingly.

Recipe level changes:
- Leather armor boots made autolearned. While I'm sure there was a sane reason for making them book-learned only, it makes them stand out as an odd exception to the other body-material boots being autolearned.
- Leather helmet shifted up to level 4 This makes them consistent with the pattern of gauntlets/boots, arm guards, helm, body armor.
- Shifted chitin gauntlets and boots up to level 4. This marks the start of the decision to split chitin's level progression from that of bone armor, with chitin chosen as the higher-level progression mostly due to higher protection. And due to chitin armor not being augmented leather armor, so one could assume it is less intuitive to craft.
- Shifted bone helmet down to level 5. This is the odd one out in the bone series. While bone gauntlets/boots naturally fit 1 level above the leather versions, and bone armor will be 1 level under chitin (see below), either way I wind up with a 1-level skill gap in the bone armor series.
- Shifted chitinous arm guards up to level 5. This shifts them to avoid the "level 5 recipe gap" thing. Their becoming 1 level above metal arm guards might have unintended consequences, but I hope it's minor.
- Shifted bone body armor down to level 6. This essentially completes the concept of splitting bone and chitin armor into separate progressions.

Mod Changes:
- Updated recipe for wyrmskin armor in Arcana mod. 
- Updated Medieval mod's overrides for boiled and plated leather armor, removed recipe override for leather armor boots.
- No Medieval Items mod updated to include leather armguards, due to blacklisting leather body armor.
- Classic roguelike classes mod: Gave rogues leather armguards, gave rangers vambraces.

Other changes:
- Right now, boiled and plated leather armor use 4 (more, when relevent) thread to combine leather body armor and armguards.
- Added new items to a few item groups where relevant.

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
Very nice!

**Comment by chaosvolt** [comment]
Ah, thank you.

As I said, almost tempted to squish things a bit, but another alternative would be adding vambraces for leather and bone. I heard vambraces being suggested before. o3o

**Comment by Rivet-the-Zombie** [comment]
>adding vambraces for leather and bone

This is good too!

**Comment by chaosvolt** [comment]
Might be an interesting idea, yes. Should I include it in this PR, or simply shift the recipe levels to account for where those items will be?

**Comment by Rivet-the-Zombie** [comment]
You can add it to this one. It's simple enough.

**Comment by chaosvolt** [comment]
Sure. And should these officially split leather/bone body/leg armor from arms like with the chitin version, or instead be separate alternatives?

**Comment by chaosvolt** [comment]
Bit short on time due to being out and about, so will check for recipes to change in mods (medieval mod I think).

**Comment by Rivet-the-Zombie** [comment]
We already have leather vests and pants; please make these a separate thing.

**Comment by chaosvolt** [comment]
Hmm? Vambraces versus arm guards, you mean?

**Comment by Rivet-the-Zombie** [comment]
Oh if that's what you meant, either or both would be fine!

**Comment by chaosvolt** [comment]
Meep. Ah right. I was basically splitting off the armor so it has arm guards. But unlike chitin arm guards, leather and bone arm guards currently have the protection, encumbrace, and coverage as their respective suits.

However, also added vambraces as something closer to other arm guard items, less encumbrance but also less protection overall.

Currently updating mods and such.

**Comment by Coolthulhu** [comment]
If you're balancing leather armor, it's probably a good idea to buff its coverage.
Leather armor doesn't provide adequate protection when it has a 15% chance of not providing any of it.

**Comment by chaosvolt** [comment]
Hmm. True, it is notable in having lower coverage than the bone and even chitin armor.

Also peculiar in that chitin arm guards do not resemble chitin body armor in terms of cover percent and encumbrance. Should that be changed?

**Comment by Coolthulhu** [comment]
> Should that be changed?

Could be. Low coverage armor is generally quite useless at the moment, as arm blocks don't enhance coverage at all.
Chitin armor should probably have no less coverage than bone equivalents, as chitin is generally found in skin-like "sheets", rarer than bone and requiring more skill to work. It can take a hit to armor values if good coverage makes it too good (avoid bumping its encumbrance, it is already high for something so light).

**Comment by chaosvolt** [comment]
Ah. Very well then.

**Comment by chaosvolt** [comment]
And thank you for the merge.

File: data/mods/RL_Classes/classes.json [hunk 1]
```diff
@@ -249,6 +250,7 @@
         "items": {
             "both": [
                 "armor_larmor",
+                "vambraces_larmor",
```
Comment (Malkeus) [inline data/mods/RL_Classes/classes.json]: typo here, vambraces_larmor should be vambrace_larmor. It makes the pink text of horror at world load.

**Comment by chaosvolt** [comment]
Ack. @_@

**Pull request merged**